store/src/server/api.rs

```diff
@@ -219,6 +273,14 @@ pub async fn serve(
 // UTILITIES
 // ================================================================================================
 
+/// Given the leaf value of the nullifier TSMT, returns the nullifier's block number.
+///
+/// There are no nullifiers in the genesis block. The value zero is instead used to signal absence
+/// of a value.
+fn nullifier_value_to_blocknum(value: Word) -> u64 {
+    value[3].as_int()
+}
```

Comment:
Should we return a `u32` from here and panic if the value is greater than `u32`? The thinking is that if the invalid value got into the database, it must be a bug somewhere in our code and one can argue that the integrity of the whole chain is suspect (and so the chain should stop operating as soon as possible).